pyproject.toml

```diff
@@ -0,0 +1,66 @@
+[tool.poetry]
+name = "sortinghat"
+version = "0.7.18"
+description = "A tool to manage identities."
+authors = [
+    "GrimoireLab Developers"
+]
+license = "GPL-3.0+"
+
+readme = "README.md"
+
+homepage = "https://chaoss.github.io/grimoirelab/"
+repository = "https://github.com/chaoss/grimoirelab-sortinghat"
+
+keywords = [
+    "development",
+    "grimoirelab"
+]
+
+packages = [
+    { include = "sortinghat" },
+    { include = "tests", format = "sdist" },
+]
+
```

Comment:
I think we missed this
```
include = [
    { path = "AUTHORS", format = "sdist" },
    { path = "NEWS", format = "sdist" },
    { path = "README.md", format = "sdist" },
]
```
https://github.com/chaoss/grimoirelab-toolkit/blob/a1fed329d25731177105fa094fa22b1029fdac1e/pyproject.toml#L25
https://github.com/chaoss/grimoirelab-toolkit/issues/44

Looks like we missed it in other tools as well

Reply:
@sduenas WDYT?

Reply:
Sure, we should add those. Where else did we miss to include those files?

Reply:
I don't see the configurations in the perceval repo (I just opened a PR https://github.com/chaoss/grimoirelab-perceval/pull/776 for it)
I will check if the other tools also miss this, and will open the PRs accordingly.